isis/src/base/apps/hist/main.cpp

```diff
@@ -66,54 +66,61 @@ void IsisMain() {
   p.Progress()->CheckStatus();
   LineManager line(*icube);
 
-  for(int i = 1; i <= icube->lineCount(); i++) {
+  for (int i = 1; i <= icube->lineCount(); i++) {
     line.SetLine(i);
     icube->read(line);
     hist->AddData(line.DoubleBuffer(), line.size());
     p.Progress()->CheckStatus();
   }
 
-  if(!ui.IsInteractive() || ui.WasEntered("TO") ) {
+  if (ui.WasEntered("TO") ) {
     // Write the results
     QString outfile = ui.GetFileName("TO");
     ofstream fout;
     fout.open(outfile.toLatin1().data());
 
-    fout << "Cube:           " << ui.GetFileName("FROM") << endl;
-    fout << "Band:           " << icube->bandCount() << endl;
-    fout << "Average:        " << hist->Average() << endl;
-    fout << "Std Deviation:  " << hist->StandardDeviation() << endl;
-    fout << "Variance:       " << hist->Variance() << endl;
-    fout << "Median:         " << hist->Median() << endl;
-    fout << "Mode:           " << hist->Mode() << endl;
-    fout << "Skew:           " << hist->Skew() << endl;
-    fout << "Minimum:        " << hist->Minimum() << endl;
-    fout << "Maximum:        " << hist->Maximum() << endl;
+    fout << "Cube:              " << ui.GetFileName("FROM") << endl;
+    fout << "Band:              " << icube->bandCount() << endl;
+    fout << "Average:           " << hist.Average() << endl;
```

Comment:
Don't these need to be arrow operators since hist is a pointer?  I wasn't able to compile using dot operators.